Tests/scs-test-runner.py

```diff
@@ -17,6 +17,7 @@
 import click
 import tomli
 
+from kaas.plugin.run_plugin import run_plugin_create, run_plugin_delete
```

Comment:
It is possible to import the stuff, but please use `subprocess.run` as with all the other helpers. This script here is more like a glorified bash script.